pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog.md

```diff
@@ -86,7 +86,7 @@ You can add the disabled attribute to  the hosted page to prevent the polling me
 
 ### Deeplinking
 
-When you publish a blog post, it’s important to be able to deep link to the post to enable features like sharing. With [`amp-live-list`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-live-list.md', locale=doc.locale).url.path}}), deep linking is possible by simply using the `id` of the blog item. For example, [https://ampbyexample.com/samples_templates/live_blog/preview/#post3](https://ampbyexample.com/samples_templates/live_blog/preview/#post3) allows you to navigate directly to the blog post with the `post3` id.
+When you publish a blog post, it’s important to be able to deep link to the post to enable features like sharing. With [`amp-live-list`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-live-list.md', locale=doc.locale).url.path}}), deep linking is possible by simply using the `id` of the blog item. For example, [https://amp.dev/documentation/examples/news-publishing/live_blog/index.html#post3]({{g.doc('/content/amp-dev/documentation/examples/documentation/Live_Blog.html', locale=doc.locale).url.path}}#post3)({{g.doc('/content/amp-dev/documentation/examples/documentation/Live_Blog.html', locale=doc.locale).url.path}}#post3) allows you to navigate directly to the blog post with the `post3` id.
```

Comment:
This should go to the preview: `{{ g.doc('/content/amp-dev/documentation/examples/previews/Live_Blog.html', locale=doc.locale) }}#post3`

Reply:
Done